index.html

```diff
@@ -1182,9 +1182,14 @@ <h2>
         </li>
 
         <li>
-Each public key MUST include <code>id</code> and <code>type</code>
-properties, and exactly one value property. The array of public keys
-MUST NOT contain duplicate entries with the same <code>id</code>.
+Each public key MAY include an <code>id</code> property. The array
```

Comment:
```suggestion
Each public key SHOULD include an <code>id</code> property. The array
```

Reply:
I'm on the fence... SHOULD or MAY? Aren't identifiers for everything in a DID Document a good idea... otherwise, it becomes difficult to refer to them... so, shouldn't we default to "hey, this is a good idea to give identifiers to all of your things"... but allow people to not do it.

Reply:
Agreed..